moabb/evaluations/splitters.py

```diff
@@ -267,3 +267,124 @@ def split(self, y, metadata):
                 yield subject_indices[train_session_idx], subject_indices[
                     test_session_idx
                 ]
+
+
+class CrossSubjectSplitter(BaseCrossValidator):
+    """Data splitter for cross subject evaluation.
+
+    This splitter enables cross-subject evaluation by performing a Leave-One-Session-Out (LOSO)
+    cross-validation on the dataset.
+
+    It assumes that the entire metainformation across all subjects is already loaded.
+
+    Unlike the `CrossSubjectEvaluation` class from `moabb.evaluation`, which manages
+    the complete evaluation process end-to-end, this splitter is solely responsible
+    for dividing the data into training and testing sets based on subjects.
+
+    .. image:: https://raw.githubusercontent.com/NeuroTechX/moabb/refs/heads/develop/docs/source/images/crosssubj.png
+        :alt: The schematic diagram of the CrossSubject split
+        :align: center
+
+    The inner cross-validation strategy can be changed by passing the
+    `cv_class` and `cv_kwargs` arguments. By default, it uses LeaveOneGroupOut,
+    which performs Leave-One-Subject-Out cross-validation.
+
+    Parameters
+    ----------
+    cv_class: cross-validation class, default=LeaveOneGroupOut
+        Inner cross-validation strategy for splitting the sessions of one subject.
+        LeaveOneGroupOut is the most common default.
+    shuffle: bool, default=False
+        Whether to shuffle the subject order. It can only be
+        used when changing the `cv_class` to a class compatible with `shuffle`.
```

Comment:
Does this make sense to have `shuffle` here? I would remove it as this is only a parameter of `cv_class`

Reply:
Okay, removed from the documentation

Reply:
It does not make sense to remove it from the documentation. Either remove it from the function or document it.